crates/ark/src/variables/r_variables.rs

```diff
@@ -357,6 +386,47 @@ impl RVariables {
         }
     }
 
+    /// Gets the value of the special variable '.Last.value' (the value of the
+    /// last expression evaluated at the top level), if show_last_value is true.
+    ///
+    /// For backward compatibility, also checks the global option "positron.show_last_value"
+    /// if it exists, but the instance variable takes precedence.
+    ///
+    /// Returns None in all other cases.
+    fn last_value(&self) -> Option<PositronVariable> {
+        // First check our instance variable
+        let show_last_value = if self.show_last_value {
```

Comment:
I see that you say

> Note that the option can be turned on during a session, but turning it off requires a restart. It's likely most users who want this will set it in their .Rprofile; we could also add a Positron setting for this.

I don't see where this occurs. During a session you can set `positron.show_last_value` and it looks like that will get used if `self.show_last_value` is `false`.

I don't see anywhere where `self.show_last_value` is persistently set to `true`, so I'm not sure how "turning it off requires a restart"

Reply:
It is set to true during tests. Turning it off requires a restart because we don't have any way to clear the value even if we aren't updating it any more.

Just typing that out makes it sound dumb so I went ahead and made it so that you can change the option on the fly w/o a restart